<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PetriButtons</class>
 <widget class="QWidget" name="PetriButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>52</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>435</width>
    <height>52</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pb_arrow">
     <property name="toolTip">
      <string>selecionar</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="imgs.qrc">
       <normaloff>:/icons/img/cursor.ico</normaloff>:/icons/img/cursor.ico</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pb_place">
     <property name="toolTip">
      <string>lugar</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="imgs.qrc">
       <normaloff>:/icons/img/place.ico</normaloff>:/icons/img/place.ico</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pb_fplace">
     <property name="toolTip">
      <string>lugar flúido</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="imgs.qrc">
       <normaloff>:/icons/img/fplace.ico</normaloff>:/icons/img/fplace.ico</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pb_itrans">
     <property name="toolTip">
      <string>transição imediata</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="imgs.qrc">
       <normaloff>:/icons/img/imtrans.ico</normaloff>:/icons/img/imtrans.ico</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pb_ttrans">
     <property name="toolTip">
      <string>transição temporizada</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="imgs.qrc">
       <normaloff>:/icons/img/ttrans.ico</normaloff>:/icons/img/ttrans.ico</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pb_arc">
     <property name="toolTip">
      <string>arco</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="imgs.qrc">
       <normaloff>:/icons/img/arc.ico</normaloff>:/icons/img/arc.ico</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="pb_inhibitor">
     <property name="toolTip">
      <string>arco inibidor</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="imgs.qrc">
       <normaloff>:/icons/img/inhibitor.ico</normaloff>:/icons/img/inhibitor.ico</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>travar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
